src/instructlab/model/serve.py

```diff
@@ -69,7 +74,7 @@ def serve(
     model_family,
     log_file,
     backend,
-    vllm_args,
+    vllm_args: list[str],
```

Comment:
@tiran I noticed that if the flag is not set, the type of `vllm_args` is a tuple not a list. Do you need to change anything to handle this case?

Reply:
I have updated the code to use `typing.Iterable[str]` in the outer layer and `list[str]` internally.